### DISCUSSION TITLE
Prefixes for kelvin (K)

### DISCUSSION THREAD

**Discussion by jfeist** [josdejong/mathjs#2821] [Ideas]
Currently, the kelvin unit does not support prefixes. However, in physics, it's very common to have to deal with either very high or very low temperatures (e.g., nK), and I don't see any reason why this should not be allowed. Could this be added?

**Comment by gwhitney** [comment]
Yeah I have no idea why Kelvin is currently set to Prefixes.NONE in src/type/unit/Unit.js -- I would suggest just filing a PR to change it to the appropriate prefixes setting.

**Reply by josdejong** [reply]
Yes indeed, we can just allow for these prefixes I think, there is no special reason for that.

**Reply by josdejong** [reply]
A fix has been published in `v11.3.2` now
